src/totalOrder.ml

```diff
@@ -1,14 +1,10 @@
-type comparison =
+(** A type represent a total order relationship *)
+
+type t =
   | Greater
   | Less
```

Comment:
I'm ok exposing the actual variant in the public type definition, but it's also important to expose values as Imm.re does to enable usage from js.